Part 1 - Personal Blog/style.css

```diff
@@ -0,0 +1,199 @@
+/* base styles */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&family=Roboto&display=swap');
+
```

Comment:
structure-wise, Rubrics asked:
1. There should be at least 3 imported files in the main CSS file.
2. Files are organized with a directory structure that separates files based on page and functionality.

Reply:
Done.